server/tests/subject_page_configs_test.py

```diff
@@ -52,8 +52,10 @@ def verify_tile(self, tile, stat_vars, msg):
 
   def test_required_fields(self):
     """Tests all configs loaded at server start"""
-    all_configs = []
-    all_configs.append(libutil.get_topic_page_config())
+    all_configs = {}
+    all_configs.update(libutil.get_topic_page_config())
```

Comment:
nit: missing key for the dict update?

Reply:
get_topic_page_config() returns a dict

Reply:
ah, poor function naming on my part then!